src/main/java/uk/gov/crowncommercial/dts/scale/cat/controller/TendersController.java

```diff
@@ -63,11 +61,11 @@ public ResponseEntity<RegisterUserResponse> registerUser(
 
     log.info("registerUser invoked on behalf of principal: {} for user-id: {}", principal, userId);
 
-    if (!Objects.equal(principal, userId)) {
-      // CON-1682-AC11
-      throw new AuthorisationFailureException(
-          "Authenticated user does not match requested user-id");
-    }
+    // if (!Objects.equal(principal, userId)) {
+    // // CON-1682-AC11
+    // throw new AuthorisationFailureException(
```

Comment:
Tom - Do we need this comments?

Reply:
No, I just commented that check out when I needed to test with a user other than the owner of the JWT - the checks must be there in the deployed version, only the authenticated user can check / register themselves - if that makes sense.